CHANGELOG.md

```diff
@@ -4,44 +4,56 @@
 
 ### New features
 
-#### Create custom width container classes
+#### Set a specific width for wrappers
 
-You can now create custom page width container classes using the `govuk-width-container` mixin. You do this by passing in the required maximum width of the container.
+You can now:
+
+- create your own Sass classes with specific widths
+- add those classes to the `.govuk-width-container` wrapper using Nunjucks
```

Comment:
The ".govuk-width-container" class is shared by the page, header and footer component, so this there is no single 'width container', could consider removing this last bullet point and clarifying the use of 'containerClasses' in the template, header, and footer.

Reply:
Clarifying `containerClasses` might be a bit involved for the release note. Something to add to the relevant bit in the variables table on the Design System page maybe?
Could we make this bullet more accurate by saying "add those classes to your layout", or something similar? Then the next bit talks about the template, header and footer specifically.

Reply:
The two new additions are:

1. the govuk-width-container mixin now can have a different width if you pass a width to it
2. the page template can have additional classes applied to it using 'containerClasses', which is consistent with the header and footer component